src/main/java/picocli/AbbreviationMatcher.java

```diff
@@ -0,0 +1,80 @@
+package picocli;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+class AbbreviationMatcher {
+    static List<String> splitIntoChunks(String command) {
```

Comment:
Let's move these methods to `CommandLine`. We should not create a separate source file for this. Sorry I was unclear.

Reply:
I knew, just make it separate temporarily to make development easier since the editor is more responsive. Will move it inside after the development is done.